code/src/components/GameQuestionsStyling.js

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import HeroPlayerInputContainer from '../assets/hero-start-page.jpg';
+
+export const DirectionText = styled.div`
```

Comment:
You could have added a background to the boxes with the directions because the opacity makes it a little hard to read the text in a good way.
Apart from that we really love your styling.